Simplified/Accounts/Library/Account.swift

```diff
@@ -425,14 +425,14 @@ class OPDS2SamlIDP: NSObject, Codable {
         do {
           self.authenticationDocument = try
             OPDS2AuthenticationDocument.fromData(serverData)
+          completion(true)
```

Comment:
Calling the `completion` before presenting the announcements, so the catalog can load in the background